app/experimenter/experiments/api/v5/inputs.py

```diff
@@ -35,6 +35,10 @@ class Meta:
         )
 
 
+class ExperimentIdInput(graphene.InputObjectType):
+    id = graphene.Int()
```

Comment:
Does this need to be its own input or can you use a subset of the ExperimentInput that includes ID?

Reply:
Figured this was better as its own input, if only because this is explicit about not needing or wanting any of the other fields